doc/modules/model_evaluation.rst

```diff
@@ -258,6 +258,122 @@ permitted and will require a wrapper to return a single metric::
+The optimal point forecast under :math:`S` is the Bayes Rule
+:math:`\hat{y} = \operatorname{argmax}_x \mathbb{E}[S(Y,x)]` (to get an
+unbiased estimate of :math:`\mathbb{E}[S(Y,x)]` for model evaluation is one
+reason to use a test set independent of the training set).
+Instead of a point forecast, one could try to issue the whole probability
+distribution :math:`F(y)` of the target variable :math:`Y`.
+This is called a probabilistic forecast.
+If you are free to choose a scoring function, you should first ask yourself,
+which functional of :math:`F(y)` you want to predict as a point forecast:
+the mean, the mode, the median, ...
+Then, a **strictly consistent** scoring function for this
+functional is to be preferred, see [G2009]_ for a definition of *strictly
+consistent* and for more details.
+The reasoning goes as follows: If the scoring function is (strictly) consistent
+for the functional at interest, this functional is the (unique) optimal point
+forecast under this scoring function.
+
+
+==================    ========================    ===============================================
+functional            scoring or loss function    property
+==================    ========================    ===============================================
+**Classification**
+mean                  Brier score                 strictly consistent
+mean                  log loss                    strictly consistent
+median                absolute error              strictly consistent
+mode                  zero-one loss               consistent (for binary classification)
+**Regression**
+mean                  squared error               strictly consistent (if finite 2nd moment)
+mean                  Poisson deviance            strictly consistent (for non-negative target y)
+mean                  Gamma deviance              strictly consistent (for positive target y)
+median                absolute error              strictly consistent
+mode                  zero-one loss               asymptotically consistent
+==================    ========================    ===============================================
+
+The zero-one loss is equivalent to the accuracy score, meaning it gives
+different score values but the same ranking.
+R² is equivalent to squared loss.
+The Brier score is the squared error for classification.
+
+An example with binary classification
+-------------------------------------
+
+Suppose you want to predict whether an apple blossom in spring becomes an
+eatable apple fruit in autumn.
+This is binary classification for :math:`Y = \text{fruit} = 1` or
+:math:`Y = \text{no fruit} = 0`.
+The good thing about binary classification is that predicting the mean of
+:math:`Y`, :math:`\operatorname{E}[Y] = p` (a result of setting the values of
+:math:`Y` to 0 and 1), is equivalent to predicting the whole distribution
+:math:`F(Y=\text{fruit}) = \operatorname{Pr}(Y=\text{no fruit}) = p`.
```

Comment:
Should the formula here be `F(Y=\text{fruit}) = \operatorname{Pr}(Y=\text{fruit}) = p` instead?
Or am I misunderstanding something?